main.go

```diff
@@ -37,50 +36,71 @@ func getConsulClient(httpAddr string, token string) *api.Client {
 	return consulClient
 }
 
-func registerConsulService(name string, frequency int, client *api.Client) {
-	fmt.Printf("Registering '%s' in consul\n", name)
-	client.Agent().ServiceRegister(&api.AgentServiceRegistration{
+func registerConsulService(name string, frequency time.Duration, client *api.Client) {
+	log.Printf("[ConsulWrapper] Registering '%s' in consul\n", name)
+	err := client.Agent().ServiceRegister(&api.AgentServiceRegistration{
 		ID:   name,
 		Name: name,
 		Check: &api.AgentServiceCheck{
 			CheckID:                        name,
 			Name:                           name,
-			DeregisterCriticalServiceAfter: strconv.Itoa(2*frequency) + "s",
-			TTL:                            strconv.Itoa(3*frequency) + "s",
+			DeregisterCriticalServiceAfter: fmt.Sprintf("%d", int64(2.1*frequency.Seconds())) + "s",
+			TTL:                            fmt.Sprintf("%d", int64(3.1*frequency.Seconds())) + "s",
 		},
 	})
+	if err != nil {
+		log.Fatal(err)
+	}
 }
 
 func deregisterConsulService(name string, client *api.Client) {
-	fmt.Printf("Deregister '%s' from consul\n", name)
-	client.Agent().ServiceDeregister(name)
+	log.Printf("[ConsulWrapper] Deregistering '%s' from consul\n", name)
+	err := client.Agent().ServiceDeregister(name)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func usage() {
+	fmt.Fprintf(os.Stderr, "Usage of %s:\n\n", os.Args[0])
+	fmt.Fprintf(os.Stderr, "\t%s [options] -service <ServiceName> -command <Command> [Args]\n\n", os.Args[0])
```

Comment:
You can use `flag.PrintDefaults()`

Reply:
The argument part would not be displayed...